plugins/arElasticSearchPlugin/lib/arElasticSearchMapping.class.php

```diff
@@ -482,4 +484,27 @@ protected function getNestedI18nObjects($languages, $nestedI18nFields)
 
     return $mapping;
   }
+
+  /**
+   * Add "sort" ES fields
+   *
+   * Add i18n "sort" field that is not tokenized, lowercase, has whitespace
+   * trimmed, punctation removed, and is ASCII folded for more natural sorting
+   */
+  protected static function addSortFields(&$nestedI18nFields, $typeProperties)
+  {
+    if (!isset($typeProperties['_attributes']['sortFields']))
+    {
+      return;
+    }
+
+    foreach ($typeProperties['_attributes']['sortFields'] as $item)
+    {
+      $nestedI18nFields[$item]['fields']['alphasort'] = array(
+        'type' => 'keyword',
+        'normalizer' => 'alphasort',
+        'include_in_all' => false,
+        'store' => false);
```

Comment:
I think `store` is set to `false` by default.

Reply:
@jraddaoui yes, looks like `store` defaults to `false` for the keyword datatype: https://www.elastic.co/guide/en/elasticsearch/reference/5.6/keyword.html

I'll update the PR